UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment.UnitTests/Services/FulfilmentDataServiceTests.cs

```diff
@@ -20,6 +21,7 @@ public class FulfilmentDataServiceTests
         private ILogger<FulfilmentDataService> _fakeLogger;
         private IFileSystemHelper _fakefileSystemHelper;
         private IConfiguration _fakeconfiguration;
+        private IFileInfo _fakeFileInfo;
 
         public FleetMangerGetAuthTokenResponseModel jwtauthUnpToken = new();
         public FleetMangerGetAuthTokenResponseModel jwtAuthJwtToken = new();
```

Comment:
Is variable jwtAuthJwtToken used? If not please remove.

Reply:
Removed